lib/chromic_pdf/supervisor.ex

```diff
@@ -393,6 +401,78 @@ defmodule ChromicPDF.Supervisor do
 
       See [`print_header_footer_template.html`](https://cs.chromium.org/chromium/src/components/printing/resources/print_header_footer_template_page.html)
       from the Chromium sources to see how these values are interpreted.
+
+      ### Wait for dynamic content
+
+      If your HTML fetches dynamic content via JavaScript, you can declare the content as "ready" by setting an attribute on a DOM element. You can wait for this attribute with the `wait_for` option.
+
+      The following example waits for the DOM element with ID `#element-id` to receive the attribute `ready-to-print` with value `true`.
+      before printing the page:
+
+          wait_for = %{
+            selector: "#element-id",
+            attribute: "ready-to-print",
+            value: "true"
```

Comment:
```suggestion
```

Also, I suggest aligning the selector here with the example HTML below, and maybe make it `#dynamic-content`?

Reply:
I knew I missed something! Sigh...

Reply:
I think we made a good call on removing the `value`, by the way. Makes the feature a lot simpler to grasp & use.